bridge_adaptivity/module/templates/module/base.html

```diff
@@ -11,6 +11,7 @@
   <title>{% block bootstrap3_title %}Bridge DASHBOARD{% endblock %}</title>
   {% bootstrap_css %}
   <link rel="stylesheet" href="{% static 'module/css/module.css' %}"/>
+  <script src="{% static 'module/js/vendor/clipboard.min.js' %}"></script>
```

Comment:
It is possible to use one of remote [storages](https://github.com/zenorocha/clipboard.js/wiki/CDN-Providers), but I experienced a page loading delay ones, so I decided include script to static...

Reply:
That is really not needed, CDN link would be fair enough. After page loading js lib will be cached.